### ISSUE TITLE
TRUST4 error

### ISSUE THREAD

**Issue by Teerapon789** [liulab-dfci/TRUST4#191]
Hi,

 After I install TRUST4 in hpc cluster, I just tried to run TRUST4 using example files and I got some errors at line 52. Do you have any suggestions?

[Thu Mar 30 23:08:54 2023] TRUST4 begins.
[Thu Mar 30 23:08:54 2023] SYSTEM CALL: /share/apps/genomics/trust4-1.0.8/fastq-extractor -t 1 -f hg38_bcrtcr.fa -o TRUST_test_toassemble  -1 example_1.fq -2 example_2.fq
[Thu Mar 30 23:08:55 2023] Start to extract candidate reads from read files.
[Thu Mar 30 23:08:55 2023] Finish extracting reads.
[Thu Mar 30 23:08:55 2023] SYSTEM CALL: /share/apps/genomics/trust4-1.0.8/trust4  -f hg38_bcrtcr.fa -o TRUST_test -1 TRUST_test_toassemble_1.fq -2 TRUST_test_toassemble_2.fq
[Thu Mar 30 23:08:55 2023] SYSTEM CALL: /share/apps/genomics/trust4-1.0.8/annotator -f human_IMGT+C.fa -a TRUST_test_final.out -t 1 -o TRUST_test  -r TRUST_test_assembled_reads.fa --airrAlignment > TRUST_test_annot.fa
Need to use -a to specify the assembly file.
system /share/apps/genomics/trust4-1.0.8/annotator -f human_IMGT+C.fa -a TRUST_test_final.out -t 1 -o TRUST_test  -r TRUST_test_assembled_reads.fa --airrAlignment > TRUST_test_annot.fa failed: 256 at /share/apps/genomics/trust4-1.0.8/run-trust4 line 52.

**Comment by mourisl** [comment]
It seems that the trust4 main executable did not generate any meaningful results. Could you please check whether TRUST_test_toassemble_1.fq was created? Is the example_1.fq file and the hg38_bcrtcr.fa file in the same path?

**Comment by Teerapon789** [comment]
Yeah, everything is in the same path. I just added -o parameter (TRUST_example) and it ran smoothly. Thank you so much

**Comment by mourisl** [comment]
Glad it works. Though even if you left the -o option unspecified, TRUST4 should use "TRUST_example" here as the default value. Just curious, what was your original command? Thank.
